README.md

```diff
@@ -21,6 +21,8 @@ $ git clone https://github.com/mirumee/saleor-platform.git --recursive --jobs 3
     - Windows/MacOS: Make sure that in Docker preferences you have dedicated at least 5 GB of memory (Preferences -> Resources -> Advanced).
     - Linux: No action required, sharing already enabled and memory for Docker engine is not limited.
 
+*Note: The latest version of Docker on Windows does not allow to change dedicated amount of memory.*
```

Comment:
I see that on newest version the memory adjustment bar is gone indeed, would be good to ensure what was the purpose (find in release notes or something). Most likely they just removed and unlimited it like it is on Linux but i would feel safer if we confirm it (in case they haven't unlimited it and just hid it somewhere else in the settings?)